test/jsx.ts

```diff
@@ -22,6 +22,7 @@ for (const { title, parser } of [
     it("should parse and print attribute comments", function () {
       check("<b /* comment */ />");
       check("<b /* multi\nline\ncomment */ />");
+      check("<b // test comment\n/>")
```

Comment:
This test passes even without the changes in `lib/printer.ts`, which makes sense because this code does not contain a `TSTypeParameterInstantiation` or `TSTypeParameterDeclaration`.

<img width="404" alt="image" src="https://github.com/benjamn/recast/assets/1938/12b106b1-a068-46d3-ae4f-d670d3b4883a">